Codo-Baggins/covid_comparisons
# Feature/comparison details logic

## What's this PR do?
- Adds logical operations to the ComparisonDetails page to dynamically create blurbs of text that compare deaths of COVID-19 to some user-selected topic. 
- Changes the way the ComparisonDetails info is being displayed.  A ComparisonCard component will be used to display various factual comparisons.  

## Where should the reviewer start?
- The reviewer should start in ComparisonDetails lines 21-36 with the function declarations.  Note that these functions will return data values that will be inserted into interpolated strings to then be displayed on the page.  

## How should this be manually tested?
- Clone down the repo
- `cd` into the cloned down directory
- Run `npm start` in the command line and navigate to the `localhost` URL that it provides you 
- Scroll down the page and click on the dropdown menus
- Select a category and then click the 'View Comparison' button
- You will be brought to a new page with the selected comparison category data
- Observe the data on the page and how no matter which category is selected, the text makes sense

## Any background context you want to provide?
- The ComparisonCards that we have created are not yet receiving any props so we are not utilizing them yet.
